tox.ini

```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist = py27-django{18,111}
+
+[testenv]
+setenv =
+    DJANGO_SETTINGS_MODULE = eventtracking.django.tests.settings
+    PYTHONPATH = {toxinidir}
+
+deps =
+    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/dev-requirements.txt
```

Comment:
Interesting.   I'll take a look at those examples.  I followed the example of edx-submissions (https://github.com/edx/edx-submissions/pull/52), which put the various Django requirements into tox.ini, and avoided cut-and-paste errors between requirements.txt and test-requirements.txt.  I agree that the name tox-requirements.txt is actually misleading, however, and might be better named release-requirements.txt or something.